### PR TITLE
Lowercase the user name

### PR DESCRIPTION
**Pull request by chaspy**
Close: https://github.com/quipper/quipper/issues/26344

### CODE REVIEW

File: kex [hunk 1]
```diff
@@ -120,7 +120,10 @@ end
 def create_pod_spec_from_original(pod_spec, app_name)
   # "kubectl whoami" would be better in some cases.
   # ref: https://github.com/kubernetes/kubernetes/pull/65884
-  runner = ENV['SUDO_USER'] || ENV['USER'] || "UNKNOWN"
+
+  # If the user name is uppercase, it will fail, so make it lowercase.
+  # ref: https://github.com/quipper/quipper/issues/26344
+  runner = ENV['SUDO_USER'].downcase || ENV['USER'].downcase || "UNKNOWN"
```
Comment (chaspy) [inline kex]:
```ruby
$ export SUDO_USER="MITSUBOSHI"
$ irb
irb(main):001:0> ENV['SUDO_USER']
=> "MITSUBOSHI"
irb(main):002:0> runner = ENV['SUDO_USER'].downcase || ENV['USER'].downcase || "UNKNOWN"
=> "mitsuboshi"
```

👍

**Comment by chaspy** [comment]
## Test

```bash
chaspy@gate-a01.qs.manage:~$ sudo cp /usr/local/bin/kex /usr/local/bin/kex.org
chaspy@gate-a01.qs.manage:~$ sudo vi /usr/local/bin/kex
chaspy@gate-a01.qs.manage:~$ diff /usr/local/bin/kex.org /usr/local/bin/kex
122,123c122,123
<   # ref: https://github.com/kubernetes/kubernetes/pull/65884
<   runner = ENV['SUDO_USER'] || ENV['USER'] || "UNKNOWN"
---
>   # ref: https://github.com/kubernetes/kubernetes/pull/6588
>   runner = ENV['SUDO_USER'].downcase || ENV['USER'].downcase || "UNKNOWN"
chaspy@gate-a01.qs.manage:~$ sudo su MITSUBOSHI
MITSUBOSHI@gate-a01:/home/chaspy$ sudo kubectl-production console -- api bin/rails c
Switched to context "qs-quipper-production-app-admin".
Context "qs-quipper-production-app-admin" modified.
shell(#<Th:0x0000560bb267cb68 run>): /home/kubernetes-app-admin-production/bin/kubectl create -o json -f -
Created pod: api-56756d48b6-kex-mitsuboshi-6jmll
Waiting for the pod to start..
Executing the specified command 'bin/rails c' on 'api-56756d48b6-kex-mitsuboshi-6jmll'...
(This container will terminate automatically in 3600 seconds.)
fatal: Not a git repository (or any of the parent directories): .git
fatal: Not a git repository (or any of the parent directories): .git
DEPRECATION WARNING: Including LoggerSilence is deprecated and will be removed in Rails 6.1. Please use `ActiveSupport::LoggerSilence` instead (called from include at /vendor/bundle/gems/rails_stdout_logging-0.0.5/lib/rails_stdout_logging/rails.rb:3)
MONGODB | Removing server mongodb-02.qs.production:27000 because it reported itself as mongodb-f03.qs.production:27000
MONGODB | Removing server mongodb-01.qs.production:27000 because it reported itself as mongodb-f01.qs.production:27000
Loading production environment (Rails 6.0.3.3)
quipper-api-rails(prod)>
```

👍 

```
quipper-api-rails(prod)> exit
Deleting pod api-56756d48b6-kex-mitsuboshi-6jmll ...
pod "api-56756d48b6-kex-mitsuboshi-6jmll" deleted

chaspy@gate-a01.qs.manage:~$ sudo cp /usr/local/bin/kex.org /usr/local/bin/kex
chaspy@gate-a01.qs.manage:~$ sudo rm /usr/local/bin/kex.org
```

File: kex (hunk 1 above)
Comment (yuya-takeyama) [inline kex]: `"UNKNOWN"` will also cause an error:

```suggestion
  runner = (ENV['SUDO_USER'] || ENV['USER'] || "UNKNOWN").downcase
```

**Review by ttokutake — APPROVED** [review]
LGTM!

**Review by MITSUBOSHI — APPROVED** [review]
LGTM 👍

**Pull request merged**